aFWall/src/main/java/dev/ukanth/ufirewall/preferences/SecPreferenceFragment.java

```diff
@@ -258,6 +280,21 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences,
             //passOption = "p" + index;
 
             //currentPosition = index;
+
+            // check if device support fingerprint, if so check if one fingerprint already existed at least
+            if(FingerprintUtil.isAndroidSupport()){
+
+                checkFingerprintDeviceSupport();
+
+            }else{
+
+                if(itemList != null) {
+
+                    itemList.setValueIndex(0);
```

Comment:
I have a solution for that.
While its preference is selecting only one choice I suggest that when user select one of the options the app should disable and clear any other options by default.
Do you support this way? :)